### PR TITLE
Prefer f-string to c-style

### LINKED ISSUE #403: Update genotype bucketing functions to use `f-string`
They're using `%` styling, and `f-string` is preferred.

### PR DESCRIPTION
**Pull request by bpblanken**
Resolves #403

### CODE REVIEW

File: luigi_pipeline/lib/model/gcnv_mt_schema.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ def samples_qs(self, start=0, end=1000, step=10):
     @row_annotation(name="samples_cn", fn_require=SeqrGenotypesSchema.genotypes)
     def samples_cn(self, start=0, end=4, step=1):
         return hl.struct(**{
-            '%i' % i: self._genotype_filter_samples(lambda g: g.cn == i)
+            f'{i}_to_{i + step}': self._genotype_filter_samples(lambda g: g.cn == i)
```
Comment (hanars) [inline luigi_pipeline/lib/model/gcnv_mt_schema.py:176]: this should just be `f'{i}'` I think

**Review by hanars — CHANGES_REQUESTED** [review]

File: luigi_pipeline/lib/model/seqr_mt_schema.py [hunk 2]
```diff
@@ -278,23 +278,23 @@ def samples_no_call(self):
     @row_annotation(fn_require=genotypes)
     def samples_num_alt(self, start=1, end=3, step=1):
         return hl.struct(**{
-            '%i' % i: self._genotype_filter_samples(lambda g: g.num_alt == i)
+            f'{i}': self._genotype_filter_samples(lambda g: g.num_alt == i)
```
Comment (bpblanken) [inline luigi_pipeline/lib/model/seqr_mt_schema.py:281]: oof and this one also flipped.

Comment (bpblanken) [reply luigi_pipeline/lib/model/seqr_mt_schema.py:281]: oh never mind

File: luigi_pipeline/lib/model/gcnv_mt_schema.py (hunk 1 above)
Comment (bpblanken) [reply luigi_pipeline/lib/model/gcnv_mt_schema.py]: this is what I get for not running the tests :(. This is even covered!

**Comment by ShifaSZ** [comment]
The changes look good. You may also want to change the logger string formats in these files. The test files also contain c-string formats which need to be updated.

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**